apps/finance/package.json

```diff
@@ -3,22 +3,20 @@
   "version": "3.0.0",
   "main": "index.js",
   "scripts": {
-    "compile": "truffle compile",
+    "postinstall": "npm run compile && npm run copy-artifacts && npm run build-app",
```

Comment:
The reason we had the prepare script previously was to avoid having to build the app.

I wonder for the sake of sanity, we should just drop building the app altogether—if someone new wants to contribute, they can manually install the app they'd like to work on rather than every app in this repo.

If we do this, can we then remove the `build-app` script?

Reply:
Yes, that's a good point. I remember taking a super long time to install the whole repo.